index.html

```diff
@@ -38,24 +38,24 @@ <h1>Im Lucca
         Im a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
         </p>
         </div>
-        <p class="let"><img src="images/LETSCONNECT.jpg" alt=""></p>
+        <p class="let"><img src="images/LETSCONNECT.jpg" alt="Hot to Connect with my website"></p>
         <div>
-            <ul class="point2">
-                <li> <a href="https://twitter.com/LuccaMicale_"><img class="twitter" src="images/Vectortw.jpg" alt=""></a></li>
-                <li> <a href="#"><img class="letteri" src="images/VectorI.png" alt=""></a></li>
-                <li> <a href="#"><img class="lettern" src="images/Vectorn.jpg" alt=""></a></li>
-                <li> <a href="#"><img class="letterm" src="images/Vectorm.jpg" alt=""></a></li>
-                <li> <a href="https://github.com/luccamicale"><img class="git" src="images/Vectorgit.jpg" alt=""></a></li>
-                <li> <a href=""><img class="hand" src="images/Vectorhnad.jpg" alt=""></a></li>
+            <ul tabindex="0" class="point2">
+                <li> <a href="https://twitter.com/LuccaMicale_"><img class="twitter" src="images/Vectortw.jpg" alt="Twitter"></a></li>
+                <li> <a href="#" ><img class="letteri"  src="images/VectorI.png" alt="Login in Linkedin"></a></li>
+                <li> <a href="#"><img class="lettern" src="images/Vectorn.jpg" alt="Login in Linkedin"></a></li>
```

Comment:
- [ ] The purpose of the text here is to give a clear description of the image, you could simply put the text here as `LinkedIn`or `LinkedIn Account`